services/appsync/src/main/resources/codegen-resources/service-2.json

```diff
@@ -1624,7 +1624,7 @@
       "members":{
         "graphqlApi":{
           "shape":"GraphqlApi",
-          "documentation":"<p>The udpated <code>GraphqlApi</code> object.</p>"
+          "documentation":"<p>The updated <code>GraphqlApi</code> object.</p>"
```

Comment:
hmm, this file is auto generated by the service model provided by the service team and the change will likely be reverted unless the service team fix it on the service model.